README.md

```diff
@@ -10,6 +10,23 @@ Github project link: https://github.com/copyblogger/genesis-sample/
 3. Activate the Genesis Sample theme.
 4. Inside your WordPress dashboard, go to Genesis > Theme Settings and configure them to your liking.
 
+## For Developers
+
+The version of [Genesis Sample on GitHub](https://github.com/copyblogger/genesis-sample/) includes tooling to check code against WordPress standards. To use it:
+
+1. Install Composer globally on your development machine. [See Composer setup steps](https://getcomposer.org/doc/00-intro.md#downloading-the-composer-executable).
+2. In the command line, change directory to the Genesis Sample folder.
+3. Type the command `composer install` to install PHP development dependencies.
+4. Type `composer phpcs` to run coding standards checks.
```

Comment:
Just FYI, this will result in `End of line character is invalid; expected "\n" but found "\r\n" (Generic.Files.LineEndings.InvalidEOLChar)`
for every file on Windows.

The standard Windows git settings force line endings into the Windows format (by setting autocrlf and savecrlf to true ).

I added 
```[sh]
[core]
        autocrlf = false
	safecrlf = false
	eol = lf
```
to my global .gitconfig to avoid this.

Reply:
Thanks for mentioning this, @christophherr. Would excluding that check be better? It seems to be quite common to ignore it: https://github.com/search?q=Generic.Files.LineEndings.InvalidEOLChar&type=Code&utf8=%E2%9C%93

Reply:
Excluding it might be the better solution. 

I've only run into problems with line endings when this phpcs rule is being used.
The standard git setting converts `lf` to `crlf` at checkout and converts `crlf` to `lf` on staging/commit. (It is possible to change that during install and in the .gitconfig)

Reply:
http://adaptivepatchwork.com/2012/03/01/mind-the-end-of-your-line/

Reply:
@GaryJones @christophherr Thanks for your feedback here. I've added to `.gitattributes` to normalize line endings to LF on checkin and checkout.

Reply:
Thank you, @nickcernis. 
Works like a charm.